dex/testing/doge/harness.sh

```diff
@@ -273,6 +276,7 @@ tmux send-keys -t $SESSION:0 C-c
 tmux send-keys -t $SESSION:1 C-c
 tmux send-keys -t $SESSION:2 C-c
 tmux send-keys -t $SESSION:3 C-c
+tmux send-keys -t $SESSION:5 C-c
```

Comment:
Also 4? If I understand correctly, this is just sending ctrl-c to the harnesses to quit whatever they are doing?

Reply:
I tried both ways .. does not seem to make much difference.

`Window #4` (not 2) is Harness Control for doge. It's the active window as we call `quit` and you could see `^C` on the command line as the only difference. 

Then we kill the session.

Reply:
Oh, 4 is the harness, then it cannot be doing something other than `./quit` I see

Reply:
Your question did make me think more .. what if you started a long running process in the harness window/tty programmatically.
In this case you would have to take care of terminating it yourself if you wanted to do that early.
You would just send it a signal!